src/TabsPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from './types/Tab';
+
+interface Props {
+  tabs: Tab[];
+}
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              className={classNames({ 'is-active': tabId === tab.id })}
+            >
+              <Link
+                to={`/tabs/${tab.id}`}
+              >
+                {tab.title}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="block" data-cy="TabContent">
+        {tabId
+          ? (tabs.find(tab => tab.id === tabId)?.content)
```

Comment:
A good decision, but usually in such conditions we must be sure that both options return some result.
If the 'tabs' array does not contain an element with the given 'tabId', it will return undefined which could lead to potential errors.
So it is better to put such things in a separate variable and have a backup version of the result. 
For example: 
```
const tabItem = tabs.find(tab => tab.id === tabId);
const tabContent = tabItem ? tabItem.content : 'No content';
```